app/lib/actions/pulp/repository/create.rb

```diff
@@ -104,6 +105,14 @@ def iso_distributor
           end
         end
 
+        def puppet_distributor
+          options = { id: "#{input[:pulp_id]}_puppet", auto_publish: true }
+          Runcible::Models::PuppetDistributor.new(input[:path],
```

Comment:
oddly i don't even see this option (absolute_path) in the pulp-admin puppet repo create command.  Maybe that is just left over in runcible from a long time ago and isn't used anymore?  Might ask the pulp team